benchmark/nested_params.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
```

Comment:
Is this necessary? :)

Reply:
unfortunately, yes. I took it from https://github.com/ruby-grape/grape/blob/master/benchmark/compile_many_routes.rb